xml/System.Runtime.Intrinsics.X86/Aes.xml

```diff
@@ -14,7 +14,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+            This class provides access to Intel AES hardware instructions via intrinsics
```

Comment:
nit: I missed this in the seed document (and am fixing locally), but instances of `Intel * hardware instructions` should probably be changed to `x86 * hardware instructions`. 

The ISAs are available on the x86 micro-architecture from multiple CPU manufacturers (such as AMD, Intel, and Via). Explicitly listing `Intel` has given some confusion as to whether or not manufacturers other than Intel are supported (they are).